packages/bot-utils/src/util/priceUtils.ts

```diff
@@ -5,6 +5,27 @@ import { fetchJson } from "ethers/lib/utils";
 import random from "random";
 import { Network, Alchemy } from "alchemy-sdk";
 
+type PriceApiResponse = {
+  open: number;
+  high: number;
+  low: number;
+  close: number;
+  volume: number;
+  date: string;
+};
+
+/* A lot of CEX does not have pair with USDC, we can convert to usd as the ratio is almost 1:1
+ * It's not critical in this application as if USDC depeg, we will over estimate the actual gas
+ * cost of an arbitrage, and that's ok.
+ **/
+export const converter = (token: string) => {
+  if (token.toUpperCase() === "USDC") {
```

Comment:
What about USDT?

Reply:
USDT is supported on CEXes but we could add that as a configuration

Reply:
Thats actually a good idea. Maybe have a configuration, where you can map from one token/symbol to another, this way it is really easy to add new tokens that are not supported by CEXes